test/e2e-v2/script/docker-compose/base-compose.yml

```diff
@@ -114,5 +114,21 @@ services:
       timeout: 60s
       retries: 120
 
+  sharding-proxy:
```

Comment:
This base docker is for some very basic components that all cases should use, I'd rather not to add this here, otherwise future developers tend to add all other components here and this file will lose its point